hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -862,7 +864,7 @@ protected void verifyBucketExists() throws UnknownStoreException, IOException {
                 if (BUCKET_REGIONS.containsKey(bucket)) {
                   return true;
                 }
-                s3Client.headBucket(HeadBucketRequest.builder().bucket(bucket).build());
+                s3AsyncClient.headBucket(HeadBucketRequest.builder().bucket(bucket).build());
```

Comment:
yeah realised I missed a join here! but doesn't matter since this isn't getting used in the benchmark

Reply:
note: this will not be contributed back to Hadoop at all. We want to do that very differently, this JIRA has details on that: https://issues.apache.org/jira/browse/HADOOP-18877